src/cli/commands/force/lightning/local/setup.ts

```diff
@@ -119,7 +86,7 @@ export class Setup extends SfdxCommand {
         if (this.flags.apilevel) {
             if (CommandLineUtils.platformFlagIsIOS(this.flags.platform)) {
                 this.logger.warn(
-                    'The apiLevel flag does not apply to the iOS platform... ignoring.'
+                    messages.getMessage('error:doesNotApplyToIOS')
```

Comment:
nit: This is a warning not an error so perhaps it should be `warning:doesNotApplyToIOS`. Also in my previous effort to take hardcoded strings to message files I didn't bother take the hardcoded strings that we log b/c they are not exposed to the user. They will be in logs / log files instead.